misc/json-to-record-converter/src/test/resources/ballerina/sample_9.bal

```diff
@@ -11,6 +11,13 @@ type Batters record {
 type ToppingItem record {
     string id;
     string 'type;
+    string color?;
+};
+
+type BaseItem record {
+    string id;
+    string 'type;
+    string color?;
 };
```

Comment:
Shall we add a test case where we have a Ballerina keyword as a json key (hence will be a Ballerina record name)? Or have we already covered that scenario?

Reply:
Yes, we have covered that scenario. You can check `sample8.json`, `sample8.ba`l test resources

Reply:
cool!